detox/src/devices/Device.js

```diff
@@ -187,7 +184,6 @@ class Device {
   async _sendPayload(key, params) {
     const payloadFilePath = this.deviceDriver.createPayloadFile(params);
     let payload = {};
-    //JS does not support {key: "asd"} JSON generation where the `key` is a variable
```

Comment:
Why was this removed? This line is important, otherwise I might be confused like before. There is nothing logical about JS.

Reply:
Then let's write a manifest about everything that's broken in JavaScript, it's not gonna be a short one...

The tests protect this specific thing from a wrong change.

Reply:
Why is this comment bothering you?